templates/event.php

```diff
@@ -42,119 +50,187 @@
+										endif;
+									elseif ( ( $attendee instanceof Attendee && $attendee->is_host() ) ) :
+											echo '<span class="event-you">' . esc_html__( 'You (host)', 'gp-translation-events' ) . '</span>';
+										else :
+											echo '<span class="event-you">' . esc_html__( 'You (event creator)', 'gp-translation-events' ) . '</span>';
+									endif;
+								endif;
+							endif;
+							?>
 						</li>
 					<?php endforeach; ?>
 				</ul>
 			</div>
-	<div class="event-details-stats">
-		<h2><?php esc_html_e( 'Stats', 'gp-translation-events' ); ?></h2>
-		<table>
-			<thead>
-			<tr>
-				<th scope="col">Locale</th>
-				<th scope="col">Translations created</th>
-				<th scope="col">Translations reviewed</th>
-				<th scope="col">Contributors</th>
-			</tr>
-			</thead>
-			<tbody>
-			<?php /** @var $row Stats_Row */ ?>
-			<?php foreach ( $event_stats->rows() as $_locale => $row ) : ?>
-			<tr>
-				<td title="<?php echo esc_html( $_locale ); ?> "><a href="<?php echo esc_url( gp_url_join( gp_url( '/languages' ), $row->language->slug ) ); ?>"><?php echo esc_html( $row->language->english_name ); ?></a></td>
-				<td><?php echo esc_html( $row->created ); ?></td>
-				<td><?php echo esc_html( $row->reviewed ); ?></td>
-				<td><?php echo esc_html( $row->users ); ?></td>
-			</tr>
-		<?php endforeach ?>
-			<tr class="event-details-stats-totals">
-				<td>Total</td>
-				<td><?php echo esc_html( $event_stats->totals()->created ); ?></td>
-				<td><?php echo esc_html( $event_stats->totals()->reviewed ); ?></td>
-				<td><?php echo esc_html( $event_stats->totals()->users ); ?></td>
-			</tr>
-			</tbody>
-		</table>
-	</div>
-	<div class="event-projects">
-		<h2><?php esc_html_e( 'Projects', 'gp-translation-events' ); ?></h2>
-		<ul>
-			<?php foreach ( $projects as $project_name => $row ) : ?>
-			<li class="event-project" title="<?php echo esc_html( str_replace( ',', ', ', $row->locales ) ); ?>">
-				<a href="<?php echo esc_url( gp_url_project( $row->project ) ); ?>"><?php echo esc_html( $project_name ); ?></a> <small> to
-				<?php
-				foreach ( explode( ',', $row->locales ) as $_locale ) {
-					$_locale = \GP_Locales::by_slug( $_locale );
+		<?php endif; ?>
+		<?php if ( ! empty( $attendees ) && ( ! $event->end()->is_in_the_past() || ( ( $attendee instanceof Attendee && $attendee->is_host() ) || current_user_can( 'manage_options' ) ) ) ) : ?>
+			<div class="event-attendees">
+				<h2><?php esc_html_e( 'Attendees', 'gp-translation-events' ); ?></h2>
+				<ul>
+					<?php foreach ( $attendees as $_user ) : ?>
+						<li class="event-attendee">
+							<a href="<?php echo esc_url( get_author_posts_url( $_user->ID ) ); ?>"><?php echo get_avatar( $_user->ID, 48 ); ?></a>
+							<a href="<?php echo esc_url( get_author_posts_url( $_user->ID ) ); ?>"><?php echo esc_html( get_the_author_meta( 'display_name', $_user->ID ) ); ?></a>
+							<?php if ( $stats_calculator->is_first_time_contributor( $event_start, $contributor->ID ) ) : ?>
```

Comment:
```suggestion
							<?php if ( $stats_calculator->is_first_time_contributor( $event_start, $_user->ID ) ) : ?>
```
This throws a notice `Trying to get property 'ID' of non-object in`